src/cluster.c

```diff
@@ -119,6 +119,14 @@ static inline int defaultClientPort(void) {
     return server.tls_cluster ? server.tls_port : server.port;
 }
 
+static int returnTLSInfo(void) {
```

Comment:
I'm not happy with the name.
Maybe `shouldReturnTlsInfo`?
And maybe some comments to explain the logic